tool/src/dart/mod.rs

```diff
@@ -1,50 +1,933 @@
-mod class;
-mod formatter;
-
 use crate::common::{ErrorStore, FileMap};
+use askama::Template;
```

Comment:
what changed in this file? Why is this diff so big?

Reply:
oh I merged `class.rs` into it because the split was kind of arbitrary and copied from cpp2. deleted now

Reply:
going to assume nothing else changed then

Reply:
Well there were some merging simplification changes, and changes to generated whitespace